backend/src/main/java/com/bakdata/conquery/models/preproc/Preprocessed.java

```diff
@@ -26,21 +27,20 @@
 	private final String name;
 	private final PPColumn primaryColumn;
 	private final PPColumn[] columns;
-	private final ImportDescriptor descriptor;
+	private final TableImportDescriptor descriptor;
 	private long rows = 0;
 	private CDateRange eventRange;
 	private long writtenGroups = 0;
-	private List<List<Object[]>> entries = new ArrayList<>();
+	private Multimap<Integer, Object[]> entries = MultimapBuilder.hashKeys().arrayListValues().build();
```

Comment:
Ich glaube hier ist `Map<Integer, List<Object[]>>` besser, da wir Die zweite Interpretation aus der Multimap-Doku benutzen und für diesen eine Multimap nicht empfohlen wird

Reply:
> A collection that maps keys to values, similar to Map, but in which each key may beassociated with multiple values. You can visualize the contents of a multimap either as amap from keys to nonempty collections of values: 
•a â†’ 1, 2 
•b â†’ 3 
... or as a single "flattened" collection of key-value pairs: •a â†’ 1 
•a â†’ 2 
•b â†’ 3 

> **Important:** although the first interpretation resembles how most multimaps are implemented, the design of the Multimap API is based on the second form.So, using the multimap shown above as an example, the size is 3, not 2,and the values collection is [1, 2, 3], not [[1, 2], [3]]. For thosetimes when the first style is more useful, use the multimap's asMap view (or create a Map<K, Collection<V>> in the first place).

Reply:
Dann sparst du dir auch den cast in Zeile 67

Reply:
Was sind deine Gedanken hierzu?